internal/util/util.go

```diff
@@ -276,11 +276,3 @@ func Int64DerefToPointer(ptr *int64, def int64) *int64 {
 	i := pointer.Int64Deref(ptr, def)
 	return &i
 }
-
-// BoolToInt converts a boolean value to an integer
-func BoolToInt(b bool) int {
```

Comment:
This is used here :) https://github.com/kaessert/provider-github/blob/6179da8d5b7e94e2891afeb93b0d32ad0d00cdf2/internal/controller/repository/repository.go#L808

Reply:
It's not used anymore: https://github.com/kaessert/provider-github/pull/20/files#diff-a961e88c47a586d25a55b6104120a8fbf3180116aa0ede211c60b2d05f614648L808

Reply:
I see, my bad :)